bootstrapvz/base/bootstrapinfo.py

```diff
@@ -55,6 +55,13 @@ def __init__(self, manifest=None, debug=False):
 		# that hint at how a command may be made available.
 		self.host_dependencies = {}
 
+		# Dictionary to store information about the image, like its resulting name
+		image_name = self.__parse_image_name(self.manifest, self.manifest_vars)
+		self.image = {
+		    'filename': image_name + '.' + self.volume.extension,
```

Comment:
Indeed. I guess it's better if we postpone the image name generation. But there's another problem here:

* In GCE, there are two distinct names: the image volume name, `disk.raw`, and the image tarball name, e.g.: `debian-8-2-jessie-v20151017.tar.gz`.
* The Vagrant plugin doesn't cares about the image volume name, as it uses [a symbolic link][1] to "rename" it to the expected `box-disk1.*`. Then, [everything is packed][2] in a `.box` file (which is also a gzipped tarball). Maybe we should use a similar approach for GCE?

[1]: https://github.com/andsens/bootstrap-vz/blob/0068cdc1a8a15061856a962045984126476b9583/bootstrapvz/plugins/vagrant/tasks.py#L134
[2]: https://github.com/andsens/bootstrap-vz/blob/0068cdc1a8a15061856a962045984126476b9583/bootstrapvz/plugins/vagrant/tasks.py#L142